src/coffeeandtacos/TacoParser.java

```diff
@@ -8,8 +8,38 @@
     public Trackable Parse(String line) {
         
         logger.logInfo("Begin Parsing");
```

Comment:
You probably don't want to log each individual parse, but instead the log when the first parsing begins.